### PR TITLE
Improvements for intelligent commands - Phase 2

### PR DESCRIPTION
**Pull request by azatsarynnyy**
### What does this PR do?
This PR polishes intelligent commands UX and UI based on the various feedbacks received during the beta
- Changes a way of creating a new command goal.
- Fixes dropdown lists opening: now whole widget's area reacts on a mouse click but not :arrow_down_small: button only
- Changes layout of command editor: all fileds are fullwidth
- Fixes reopening command editors after refreshing the browser or reopening the workspace
- Adds shortcuts for default "Run" and "Debug" commands
- Adds tooltip for process ID in command toolbar

### What issues does this PR fix or reference?
#4442 

#### Changelog
Intelligent commands UX and UI improvements:
- Changed flow of new command goal creation
- Processes and Previews dropdown lists made fully clickable
- Command editor made fullwidth
- Command editors now are restorable as well as text editors
- Added shortcuts for the default "Run" and "Debug" commands: Alt+R and Alt+D respectively
- Added tooltip for process ID in commands toolbar

#### Release Notes
N/A
Will be released with https://github.com/eclipse/che/pull/4389

#### Docs PR
https://github.com/eclipse/che-docs/pull/168

### CODE REVIEW

**Comment by slemeur** [comment]
@azatsarynnyy : Could you please add the proper changelog (you can have multiple lines)

**Review by slemeur — APPROVED** [review]

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/command/toolbar/commands/button/GoalButton.java [hunk 1]
```diff
@@ -75,6 +95,34 @@ private void setTooltip(String newTooltipText) {
...
-        tooltip = Tooltip.create((Element)getElement(), BOTTOM, MIDDLE, newTooltipText);
+        final DivElement divElement = Elements.createDivElement();
+        divElement.setInnerText(newTooltipText);
+
+        if (keyBinding != null) {
+            final String hotKey = getShortcutText(keyBinding);
+            if (hotKey != null) {
+                SpanElement spanElement = Elements.createSpanElement();
+                spanElement.getStyle().setMarginLeft("5px");
+                spanElement.getStyle().setColor("#aaaaaa");
```
Comment (vparfonov) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/command/toolbar/commands/button/GoalButton.java:106]: How it will be on white theme?

Comment (slemeur) [reply ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/command/toolbar/commands/button/GoalButton.java:106]: White theme is not handled yet.
Will be part of that issue: https://github.com/eclipse/che/issues/4590

**Review by vparfonov — APPROVED** [review]

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/command/toolbar/commands/button/GoalButton.java (hunk 1 above)
Comment (azatsarynnyy) [reply ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/command/toolbar/commands/button/GoalButton.java:106]: BTW it looks good on both themes

**Pull request merged**